core/chains/evm/client/node.go

```diff
@@ -61,6 +61,10 @@ var (
 		Name: "evm_pool_rpc_node_calls_success",
 		Help: "The approximate total number of successful RPC calls for the given RPC node",
 	}, []string{"evmChainID", "nodeName"})
+	promEVMPoolSendTransactionTiming = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "evm_pool_rpc_node_send_tx_timing",
+		Help: "The duration of a SendTransaction call in nanoseconds",
+	}, []string{"evmChainID", "nodeName", "rpcHost", "isSendOnly"})
```

Comment:
Should we also add a label for whether or not the call was successful?

Reply:
That's a good idea, will do that!